hazelcast/src/main/java/com/hazelcast/internal/config/ConfigUtils.java

```diff
@@ -44,7 +44,18 @@ public final class ConfigUtils {
     private ConfigUtils() {
     }
 
-    public static <T> T lookupByPattern(ConfigPatternMatcher configPatternMatcher, Map<String, T> configPatterns,
+    /**
+     * Lookup {@code itemName} in {@code configPatterns}, according to matching rules applied by
+     * {@code configPatternMatcher}.
+     * <p/>
+     * Probably this is NOT the method you are looking for: it returns the object located in {@code configPatterns}
```

Comment:
![obi-wan-kenobi-these-are-not-the-droids](https://user-images.githubusercontent.com/8236272/192972824-fb4dfa29-be4a-44c3-a213-e11b8292ac2b.gif)